funnel/templates/index.html.jinja2

```diff
@@ -1,7 +1,8 @@
 {% extends "layout.html.jinja2" %}
 {% block title %}{{ config['SITE_TITLE'] }}{% endblock %}
 {% from "baseframe/components.html.jinja2" import faicon %}
-{%- from "macros.html.jinja2" import calendarwidget, saveprojectform, featured_section, upcoming_section, open_cfp_section, all_projects_section, past_projects_section %}
+{%- from "macros.html.jinja2" import calendarwidget, saveprojectform, featured_section, upcoming_section, open_cfp_section, all_projects_section %}
+{%- from "js/past_projects.js.jinja2" import past_projects_section %}
```

Comment:
These templates can be moved back into the parent folder.

Subfolders have a lookup cost in Jinja2 because Flask blueprints also appear like subfolders (like the `baseframe/components` above). Every time a template is imported from a subfolder, Jinja2 will scan through _both_ blueprints and filesystem folders looking for a match.

Reply:
(It is possible this extra lookup cost is negligible. Worth measuring.)